packages/frontend/src/components/Auction/UserActionSection.tsx

```diff
@@ -30,7 +30,9 @@ export const UserActionSection = () => {
   const Content = UserActions[state]
   return (
     <Wrapper>
-      <Content />
+      <div>
+        <Content />
```

Comment:
The height of the user action panel to the right is not constant. This results in views like this:
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/3901896/177990227-0fa1b504-953e-46c3-ac3e-fce3e62512ca.png">